ext/opcache/jit/README.md

```diff
@@ -24,5 +24,116 @@ was almost 100 times slower, making it prohibitively expensive to use.
 [The unofficial DynASM Documentation](https://corsix.github.io/dynasm-doc/tutorial.html)
 has a tutorial, reference, and instruction listing.
 
-`zend_jit_x86.dasc` gets automatically converted to `zend_jit_x86.c` by the bundled
+In x86 builds, `zend_jit_x86.dasc` gets automatically converted to `zend_jit_x86.c` by the bundled
 `dynasm` during `make`.
+
+In arm64 builds, `zend_jit_arm64.dasc` gets automatically converted to `zend_jit_arm64.c` by the bundled
+`dynasm` during `make`.
+
+Running tests of the JIT
+------------------------
+
+Then, to test the JIT, e.g. with opcache.jit=tracing, an example command
+based on what is used to test in Azure CI:
+
+```
+make test TESTS="-d opcache.jit_buffer_size=16M -d opcache.enable=1 -d opcache.enable_cli=1 -d opcache.protect_memory=1 -d opcache.jit=tracing --repeat 2 --show-diff -j$(nproc) ext/opcache"
+```
+
+- `opcache.jit_buffer_size=16M` enables the JIT in tests by providing 16 megabytes of
+  memory to use with the JIT to test with.
+- `opcache.protect_memory=1` will detect writing to memory that is meant to be
+  read-only, which is sometimes the cause of opcache bugs.
+- `--repeat 2` is optional, but used in CI since some JIT bugs only show up after processing a
+  request multiple times, e.g. due to a bug in the engine or bad data in the
+  runtime cache of the interpreter.
```

Comment:
The actual reason is that the first request compiles the trace and the second executes it.